pyfar/dsp/dsp.py

```diff
@@ -143,6 +143,51 @@ def wrap_to_2pi(x):
     return x
 
 
+def linear_phase(signal, group_delay=None):
+    """
+    Calculate linear phase signal.
+
+    The linear phase signal is computed as
```

Comment:
This description can be a bit confusing, as one might expect that this function calculates sets the phase to "only" the already existing linear phase part of the singal, i.e. a already pre-existing constant group delay.

Reply:
Set the phase to a linear phase with group delay ...